core/src/main/java/brooklyn/entity/rebind/PeriodicDeltaChangeListener.java

```diff
@@ -164,14 +163,12 @@ public void remove(BrooklynObject instance) {
     private final PersistenceExceptionHandler exceptionHandler;
     
     private final Duration period;
-    
-    private final AtomicLong writeCount = new AtomicLong();
-    
+        
     private DeltaCollector deltaCollector = new DeltaCollector();
 
     private volatile boolean running = false;
 
-    private volatile boolean stopped = false;
+    private volatile boolean stopping = false, stopCompleted = false;
```

Comment:
`stopping`, `stopCompleted` and `running` could be merged into a single enum to manage state easier, can be done in a future PR.